gp2gp_adaptor_response_docs.yaml

```diff
@@ -0,0 +1,557 @@
+
+            This is inserted into the GP2GP placeholder document as the original filename and suffix. So should be of the form filename.suffix, e.g. referral.txt'
+        EhrReceivedAcknowledgement:
+            type: object
+            properties:
+                received:
+                    type: string
+                    format: date-time
+                    nullable: False
+                    description: 'The date and time (ISO-8601) the acknowledgment was received'
+                conversationClosed:
+                    type: string
+                    format: date-time
+                    nullable: True
+                    description: 'If the acknowledgement ended the transfer, the date and time (ISO-8601) the conversation was closed can be found here'
+                errors:
+                    type: array
+                    items:
+                        $ref: '#/components/schemas/ErrorDetails'
+                    nullable: True
+                    description: 'An array of errors which will be populated in the case of a negative / rejected acknowledgement'
+                messageRef:
+                    type: string
+                    nullable: False
+                    description: 'The Message ID of the message that has been acknowledged'
+        EhrStatus:
+            type: object
+            properties:
+                attachmentStatus:
+                    type: array
+                    items:
+                        $ref: '#/components/schemas/AttachmentStatus'
+                    description: 'An array of statuses for each document sent during the transfer'
+                    nullable: False
+                migrationLog:
+                    type: array
+                    items:
+                        $ref: '#/components/schemas/EhrReceivedAcknowledgement'
+                    nullable: False
+                    description: 'An array containing details of acknowledgments received during the transfer'
+                migrationStatus:
+                    type: string
+                    enum:
+                        - COMPLETE
+                        - COMPLETE_WITH_ISSUES
+                        - FAILED_NME
+                        - FAILED_INCUMBENT
+                        - IN_PROGRESS
+                    nullable: False
+                    description: 'The current state of the transfer, a status of COMPLETE_WITH_ISSUES is given if placeholder documents were sent'
+                originalRequestDate:
+                    type: string
+                    format: date-time
+                fromAsid:
+                    type: string
+                toAsid:
+                    type: string
+        ErrorDetails:
+            type: object
+            properties:
+                code:
+                    type: string
+                display:
+                    type: string
```

Comment:
A bit sparse, but not the end of the world. Nullable: false would be good.

Reply:
I've added it